Readme.md

```diff
@@ -217,3 +217,9 @@ List of user folders:
 Custom textures have to be placed in the user directory under
 `Load/Textures/[GameID]/`. You can find the Game ID by right-clicking a game
 in the ISO list and selecting "ISO Properties".
+
+## Downloading Pull Request (PR) Builds
+
+PR builds can be downloaded for testing purposes. In the link below, replace "XXXX" with the PR number to download it.
+
+http://<i></i>dl.dolphin-emu.org/prs/pr-XXXX-dolphin-latest-x64.7z
```

Comment:
I doubt the `<i></i>` should be there.

Reply:
That was the markdown to remove the hyperlink. Do you know a better way to do that? We don't want people complaining that http://dl.dolphin-emu.org/prs/pr-XXXX-dolphin-latest-x64.7z gives a 404 error. The preview of readme.md displays properly, see https://github.com/Ebola16/dolphin